### PR TITLE
V2.28.0

### PR DESCRIPTION
**Pull request by marinaserranomontes**
### Contributing checklist
- [ ] Code must follow existing styling conventions
- [ ] Added a descriptive commit message
- [ ] Sample apps updated if needed

### CODE REVIEW

File: @types/index.d.ts [hunk 1]
```diff
@@ -511,6 +511,9 @@ declare module "opentok-react-native" {
 
     /**
      * Sets video transformers for the publisher (or clears them if passed an empty array).
+     * To use this method, add `pod 'VonageClientSDKVideoTransformers'` to your
+     * Podfile and add `implementation "com.vonage:client-sdk-video-transformers:2.27.2"`
```
Comment (pvenum) [inline @types/index.d.ts:515]: 2.28.0

**Review by v-kpheng — APPROVED** [review]
LGTM! 💪 🚀

File: README.md [hunk 2]
```diff
@@ -121,6 +127,12 @@ If you try to archive the app and it fails, please do the following:
    <uses-permission android:name="android.permission.CAMERA" tools:node="remove"/>
    ```
 
+3. If your app will use the `OTPublisher.setVideoFilters()` method, you need to include the following in your app/build.gradle file:
```
Comment (jintgeorge) [inline README.md:130]: To confirm, setVideoFilters() is the name used in react code for setVideoTransformers()?

**Review by jintgeorge — APPROVED** [review]
LGTM. Left a minor comment.

**Review by v-kpheng — APPROVED** [review]
LGTM! 💪 🚀

**Pull request merged**